picamera2/picamera2.py

```diff
@@ -187,9 +190,9 @@ def find_tuning_algo(tuning: dict, name: str) -> dict:
 
     @staticmethod
     def global_camera_info() -> list:
-        """
-        Return Id string and Model name for all attached cameras, one dict per camera,
-        and ordered correctly by camera number. Also return the location and rotation
+        """Return Id string and Model name for all attached cameras, one dict per camera.
+
+        Ordered correctly by camera number. Also return the location and rotation
```

Comment:
The original seems to read more naturally to me...

Reply:
I could look at reverting these comment changes if you'd like and add an exception, so we don't have to have
a summary line?

Reply:
Ah, I guess I hadn't understood what's happening. It expects a "summary line" as the first one in the comment, and then a blank line and then the rest? Maybe we need to edit these just a bit so they don't become a bit ungrammatical, but I'm happy to adopt this going forward.